Formula/ucloud.rb

```diff
@@ -1,8 +1,17 @@
 class Ucloud < Formula
   desc "The official tool to managment your ucloud services"
   homepage "https://www.ucloud.cn"
-  url "https://ucloud-sdk.dl.ufileos.com/ucloud-cli-0.1.2.tar.gz"
-  sha256 "f41aaa6bf8063b3ccd0c5e41d3dc90f46f9fb45066513da8e306456f1f332071"
+  url "https://github.com/ucloud/ucloud-cli.git",
+      :tag => "0.1.3",
+      :revision => "907f6391705400ec5cb59353e38834cf0193db3b"
+
+  bottle do
```

Comment:
Don't touch (and don't add) bottle blocks please

Reply:
Sorry, It is a mistake when execute 'git rebase -i upstream/master'.  fixed.